lib/assets/javascripts/cartodb/common/dialogs/delete_column/delete_column_view.js

```diff
@@ -57,7 +56,7 @@ module.exports = BaseDialog.extend({
 
   ok: function() {
     this._panes.active('loading');
-    this.table.deleteColumn(this.column);
+    this.table.deleteColumn(this.options.column);
```

Comment:
`this.column` is no longer always the original column name.  `this.deleteColumn` needs the original column name as an input to function correctly.  Since `this.options.column` will always be the original column name and not an alias it is best to use this value as an input.  I made no changes to the model's `options`.